examples/nextjs-flow/app/hooks/useFlowTranscript.ts

```diff
@@ -0,0 +1,209 @@
+import {
+  type FlowIncomingMessageEvent,
+  useFlowEventListener,
+} from '@speechmatics/flow-client-react';
+import { useCallback, useMemo, useState, useEffect } from 'react';
+import { Partials, type Word, getWords } from './partials';
+
+export type TranscriptGroup =
+  | {
+      type: 'speaker';
+      speaker: string;
+      data: Word[];
+    }
+  | {
+      type: 'agent';
+      data: AgentResponse[];
+    };
+
+export function transcriptGroupKey(transcriptGroup: TranscriptGroup) {
+  return `${transcriptGroup.type === 'agent' ? 'agent' : transcriptGroup.speaker}-${transcriptGroup.data[0].startTime ?? 0}-${transcriptGroup.data.at(-1)?.endTime ?? '?'}`;
```

Comment:
Is biome applying to this directory? This line seems very long

Reply:
I think so as it complains about other issues (like the unnecessary dependency) - I think this is the same length in the other repo?